site/learn/BestPractice-Authorization.md

```diff
@@ -3,6 +3,56 @@ title: Authorization
 layout: ../_core/DocsLayout
 category: Best Practices
 permalink: /learn/authorization/
+next: /learn/serving-over-http/
 ---
 
-WIP
+> Delegate authorization logic to the business logic layer
+
+Authorization is a type of business logic that describes whether a given user/session/context has permission to perform an action or see a piece of data. For example:
+
+*“Only authors can see their drafts”*
+
+Enforcing this kind of behavior should happen in the [business logic layer](/learn/thinking-in-graphs/#business-logic-layer). It is tempting to place authorization logic in the GraphQL layer like so:
+
+```javascript
+var postType = new GraphQLObjectType({
+  name: ‘Post’,
+  fields: {
+    body: {
+      type: GraphQLString,
+      resolve: function (post, args, context, rootValue) {
```

Comment:
wow, i checked the airbnb js style guide and they agree with you. turns out i'm the crazy one